### DISCUSSION TITLE
湖北工业大学

### DISCUSSION THREAD

**Discussion by cwdoo** [CollegesChat/university-information#114] [University]
* ![image](https://user-images.githubusercontent.com/59089439/134634373-baf0c808-890c-438d-b627-3a485ae94f87.png)
*

**Comment by cwdoo** [comment]
![QQ图片20210924152110](https://user-images.githubusercontent.com/59089439/134634594-e05c4936-5ac3-44d6-9ca2-ebcf95beebd0.jpg)

**Comment by cwdoo** [comment]
![image](https://user-images.githubusercontent.com/59089439/134634739-3cdbd6fe-dedf-42d5-bca6-38927e097b33.png)

**Comment by MRealLonG** [comment]
你意欲何为？
